src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -71,3 +71,23 @@
     }
   }
 }
+
+.#{$prefix}-top-cropped-subtitle {
+  top: 0 !important;
```

Comment:
Make this alphabetically ordered

Reply:
Obsolete - using inline css instead of classes now:
[Use inline css instead of classes for cropped captions](https://github.com/bitmovin/bitmovin-player-ui/pull/538/commits/56a168378e3bc33a86282816f5ef17d73c4a3653)